mayastor/tests/replica_snapshot.rs

```diff
@@ -99,41 +121,59 @@ fn replica_snapshot() {
     test_init!();
 
     Reactor::block_on(async {
-        create_nexus().await;
-        bdev_io::write_some(NXNAME).await.unwrap();
+        create_nexus(0).await;
+        bdev_io::write_some(NXNAME, 0, 0xff).await.unwrap();
         custom_nvme_admin(0xc1)
```

Comment:
Not part of this change, but it would be nice to replace 0xc1 with a symbol

Reply:
I've opted for a code comment.